app/services/transitions.ts

```diff
@@ -151,6 +153,11 @@ export class TransitionsService extends StatefulService<ITransitionsState> {
     this.sceneCollectionsService.collectionWillSwitch.subscribe(() => {
       this.disableStudioMode();
     });
+
+    const establishedContext = this.videoSettingsService.establishedContext.subscribe(() => {
+      if (!this.studioModeTransition) this.createStudioModeTransition();
+      establishedContext.unsubscribe();
+    });
```

Comment:
i think we could use an explanatory comment here since it's not necessarily immediately understood behavior

Reply:
Sounds good